feature-utils/poly-look/src/react-components/overlays/screen.css

```diff
@@ -0,0 +1,9 @@
+.screen {
```

Comment:
I hate to be that person, but could you prefix `screen` with `poly`?  Same for `standard-layout`😓 I know we currently  don't really do this consistently, but in the future we want to, so it will be a bit less work for whomever has to implement it (probably me, kinda volunteered to take a look at some point 😹).

Reply:
Hmm I don't think we _want_ to prefix everything with poly. It's just super redundant. I think a Card can be a Card and doesn't need to be a polyCard and a screen can be a screen. Some things it makes sense like for polyButton, but you wouldn't say polyIconButton

Reply:
I think @g3o314 refers to [this comment](https://github.com/polypoly-eu/polyPod/pull/821#discussion_r858590305) by @fhd, and it's mainly related to the CSS classes, not the name of the classes itself.

Boils down to: so far we've used, more or less consistently, name of CSS classes and components are the same. However, going forward, it apparently will go the way @g3o314 says.

Reply:
Ahh good to know, will implement!

Reply:
@JJ is right. I'm sorry I didn't make it more explicit. It's about the CSS classes and avoiding polluting the global namespace.

Reply:
@g3o314 Na totally fine, you couldn't know that I didn't know :D